### DISCUSSION TITLE
Unable to find component

### DISCUSSION THREAD

**Discussion by marcwitteveen** [Power-Components/livewire-powergrid#1246] [General]
After installing powergrid I created a new table, however when I copy the livewire tag to my blade file I am receiving the following error:

Livewire \ Exceptions \ ComponentNotFoundException
Unable to find component: [breed_group_table]

Everything is installed on the default locations.

Please advice.

**Comment by marcwitteveen** [comment]
Uninstalled powergrid and reinstalled it again and then it worked.

**Comment by djkml001** [comment]
still have the same problem

**Comment by shakeel2717** [comment]
i'm also facing this issue? please let me know if you found any sultion, the powergrid table working 100% fine on local dev with php artisan serve command, but on live server, it's not working, i'm always facing error unable to find component, but if i create manually another livewire component, then it's rendered successfully without any error, only powergrid component are not recognized by livewire, how i can fix this?
